conf/default/login.conf

```diff
@@ -65,3 +65,6 @@ msg_server_ip: 127.0.0.1
 
 #Logging of user IP address to database (true/false)
 log_user_ip: false
+
+#New accounts may be created? (true/false)
```

Comment:
maybe rephase to specify that creation via the loader/lobby is what is disabled.

Reply:
Is there another method for account creation? I know in theory they can be added to the database directly, linking into some web front-end but that wouldn't involve the server and I assume anyone doing that level of integration wouldn't be confused by the comment here.

I was aiming for it to be simple for newer users who may not know whether they're missing another setting somewhere to disable any theoretical "other types" of logins. Do you have a specific phrase you'd prefer?

Reply:
Answered your own question: I was thinking people might mistake it for preventing any new accounts, when in reality it will be used mainly to steer people toward one of those webpage signups you just referenced.

> I assume anyone doing that level of integration wouldn't be

haha oh man...If you only knew the things I've had to explain :(

Reply:
I guess I'll suggest:
`#Allow account creation via the loader (true/false)`

Reply:
Sounds clear enough, I like it. 

Changed.